scripts/write_to_db.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+import numpy as np
+from spreadsheets_helper import read_spreadsheet
+from database_helper import connect_to_db, update_row_orm, close_connection
+
+def main():
+    SCOPES = ['https://www.googleapis.com/auth/spreadsheets']
+    SPREADSHEET_ID_INPUT = '1RZDZTswZxI-smwnyi8Oh5ck092hgl_CBxjtQURrczBs'
+    SPREADSHEET_RANGE ='A1:Z10'
+    credentials_file_name = 'credentials.json'
+    DATABASE_URL = ''
+
+    df = read_spreadsheet(SPREADSHEET_ID_INPUT, SPREADSHEET_RANGE, SCOPES)
+    df['id'] = df['id'].astype(int)
+    db_session = connect_to_db(DATABASE_URL)
+
+    for index, row in df.iterrows():
+        store_name = row['name']
+        flagged = row['flag']
+        deleted = row['deleted']
+        reviewed = row['Reviewed']
+        useful_info = row['availabilityInfo'].strip().lower() if row['availabilityInfo'] else ''
+        safety_info = row['safetyInfo'].strip().lower() if row['safetyInfo'] else ''
+
+        update_dict = {
+            'flag': row['flag']
+        }
+
+        # If deleted is marked true, then it has been reviewed
+        if deleted == 'TRUE':
```

Comment:
why is this a string and not boolean? because coming from the google sheet? does case matter? if so, this should be variable between where it's written and where it's checked (here)

Reply:
Yes, all of this gets messed up because of how G Sheets is, and everything there is text. Case matters because of how it's written there. What do you mean should be variable? Should handle variances in case?